app/views/user_mailer/in_person_ready_to_verify.html.erb

```diff
@@ -27,6 +27,7 @@
         <p><%= t('in_person_proofing.process.barcode.info') %></p>
         <%= render BarcodeComponent.new(
               barcode_data: @presenter.enrollment_code,
+              barcode_image_url_method: -> { @presenter.barcode_image_url },
```

Comment:
On second thought.... what does this block give us? Since we pass the code in to the `BarcodeComponent`, the component has all the info it needs to generate barcode images, right? So we could pass in a `mode: :href | :data_url` and the component could choose how to render?

Reply:
We could probably drop the proc. Regarding the modes idea, I think it could be inferred if the component was provided a URL option?